docker: publish docker images to docker hub (#896)
kopia/kopia: tools/docker/README.md

> # this is a temporary from which we produce docker images from pre-built binaries in dist/ directory
> # this avoids having to send the entire codebase to docker daemon.

Reviewer: ```suggestion
This is a directory for staging pre-built binaries from the  `dist/` directory to prevent sending the entire codebase to the docker daemon.
```

Independent of this PR, it may make sense to add a `.dockerignore` file for other scenarios such as building the kopia binaries with `docker build`. Anyway, that's independent.
